### PR TITLE
feat: incident manager provider

### LINKED ISSUE #804: New Provider - AWS Incident Manager
AWS incident manager is a native AWS Service that is very similar to pager duty. Every incident will have its own ARN(amazon resource name).

When an alert triggers, we can create an incident in the alert manager. While sending the resolved we need to run the API call to the incident manager and pass the ARN of that incident. So somewhere the Keep should maintain the track of ARN id so it can send the resolved status to the incident manager.

### PR DESCRIPTION
**Pull request by mohanish2504**
/closes #804
/claim #804

### CODE REVIEW

File: keep/providers/incident_manager_provider/incident_manager_provider.py [hunk 1]
```diff
@@ -0,0 +1,439 @@
+"""
```
Comment (shahargl) [inline keep/providers/incident_manager_provider/incident_manager_provider.py:1]: Add:
1. Documentation on how to connect with the provider
2. Provider icon

**Review by shahargl — CHANGES_REQUESTED** [review]

**Comment by mohanish2504** [comment]
@shahargl @talboren sorry for dealyed response will add the remaing part

**Comment by shahargl** [comment]
@mohanish2504 can you add `incidentmanager-provider.mdx` to mint.json so it will show up? also, can you add the logo to the README? I'll merge after.

**Review by shahargl — CHANGES_REQUESTED** [review]
see comments

**Comment by mohanish2504** [comment]
Hey @shahargl been busy since last few days, will try to push the commit by thursday at max.

**Comment by mohanish2504** [comment]
@shahargl can you have a look.

Also hope you guys are doing well.

**Comment by shahargl** [comment]
@mohanish2504 yo - is this still relevant?

**Comment by mohanish2504** [comment]
any more changes required here? I have pushed the changes you left for review last time. Let me know if you want any other change.

I just saw the conflicts will resolve those too.

File: docs/mint.json [hunk 2]
```diff
@@ -126,7 +126,10 @@
             "providers/documentation/telegram-provider",
             "providers/documentation/trello-provider",
             "providers/documentation/twilio-provider",
+            "providers/documentation/zenduty-provider",
```
Comment (shahargl) [inline docs/mint.json:129]: ?

File: docs/mint.json [hunk 3]
```diff
@@ -126,7 +126,10 @@
...
+            "providers/documentation/resend-provider",
```
Comment (shahargl) [inline docs/mint.json:130]: ?

**Comment by shahargl** [comment]
can you just fix the mint.json and remove the redundant logo? (only the -icon is needed)

**Comment by mohanish2504** [comment]
> t logo? (only the -icon is needed)

do you mean to also remove this?
`"providers/documentation/incidentmanager-provider"`

**Comment by shahargl** [comment]
lgtm, merging @mohanish2504

**Pull request merged**